### PR TITLE
1139 create an environment for accessibility audit

### PR DESCRIPTION
**Pull request by felixtheflex**
### Context

Create an environment for accessibility audit

### Changes proposed in this pull request

Introduce accessibility audit environment\app

### CODE REVIEW

File: terraform/workspace-variables/audit.tfvars [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# PaaS
+paas_app_environment       = "audit"
+env_config                 = "audit"
+paas_space_name            = "bat-qa"
+paas_postgres_service_plan = "tiny-unencrypted-11"
+paas_redis_service_plan    = "tiny-4_x"
```
Comment (saliceti) [inline terraform/workspace-variables/audit.tfvars:6]: Not 5?

File: config/settings/audit.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# The url for this app, also used by `dfe_sign_in`
+base_url: https://audit.register-trainee-teachers.education.gov.uk
```
Comment (vigneshmsft) [inline config/settings/audit.yml:2]: is this going to be a custom domain? or this needs changing to a cloudapps url?

**Review by vigneshmsft — APPROVED** [review]

File: config/settings/audit.yml (hunk 2 above)
Comment (felixtheflex) [reply config/settings/audit.yml:2]: they wanted to be customdomain.....

File: terraform/workspace-variables/audit.tfvars [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+paas_space_name            = "bat-qa"
+paas_postgres_service_plan = "tiny-unencrypted-11"
+paas_redis_service_plan    = "tiny-5_x"
+paas_app_start_timeout     = "180"
+paas_web_app_instances     = 1
+paas_web_app_memory        = 512
+paas_worker_app_instances  = 1
+paas_worker_app_memory     = 512
+paas_worker_app_stopped    = false
+statuscake_alerts                = {}
```
Comment (saliceti) [inline terraform/workspace-variables/audit.tfvars:13]: misaligned =
run terraform fmt?

**Review by saliceti — APPROVED** [review]

**Review by misaka — APPROVED** [review]
Nice, now it's clear we're replacing `pin` with `audit`.

**Comment by felixtheflex** [comment]
[Create an environment for accessibility audit](https://trello.com/c/mmKmJxQ2/1139-create-an-environment-for-accessibility-audit)

**Pull request merged**